### ISSUE TITLE
triangle: Provide parameters in test failure message.

### ISSUE THREAD

**Issue by Insti** [exercism/xruby#653]
The current message is:
```
  1) Failure:
TriangleTest#test_equilateral_triangle_sides_may_be_floats [ruby/triangle/triangle_test.rb:36]:
Expected 'true', triangle is equilateral.
```

I would like to see something like:
```
Expected 'true', triangle [0.5, 0.5, 0.5] is equilateral.
```
So it is possible to see from the failure message which triangle is being tested.

**Comment by Insti** [comment]
Start here: https://github.com/exercism/xruby/blob/master/exercises/triangle/.meta/generator/triangle_case.rb#L27

**Comment by deohtee** [comment]
Hello! Would love to try to tackle this as my first contribution!

**Comment by Insti** [comment]
Awesome @deohtee. 

If you have any questions, either ask them here or come to the [xruby Gitter chat room](https://gitter.im/exercism/xruby).

**Comment by Insti** [comment]
Fixed by #659 Thanks @deohtee
